src/identity/email-password-identity-provider/default-views/select_account.ejs

```diff
@@ -0,0 +1,7 @@
+<form autocomplete="off" action="/interaction/<%= uid %>/continue" method="post">
```

Comment:
Any reason `autocomplete` is off?

Reply:
Not to my knowledge. There aren't any autocompletable elements on this page.